pages/print_labels/source_plates.vue

```diff
@@ -95,7 +95,6 @@ export default {
       status: statuses.Idle,
       alertMessage: '',
       printer: 'heron-bc1',
-      numberOfBarcodes: 1,
```

Comment:
should this be numberOfBarcodes: '1' ?

Reply:
This page doesn't use this value at all, so I removed it.